File: order-service/src/main/java/com/ecommerce/orderservice/domain/dto/res/OrderResponse.java
package com.ecommerce.orderservice.domain.dto.res;

import com.ecommerce.orderservice.domain.entity.OrderStatus;
import lombok.*;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class OrderResponse {
    private Long orderId;
    private Long totalPrice;
    private OrderStatus orderStatus;
    private LocalDateTime createdAt;
    private List<OrderItemResponse> itemList;

}

File: review-service/src/main/kotlin/com/ecommerce/reviewservice/client/order/dto/res/OrderResponse.kt
package com.ecommerce.reviewservice.client.order.dto.res

import com.ecommerce.reviewservice.client.order.dto.OrderStatus
import java.time.LocalDateTime

data class OrderResponse(
    val orderId: Long,
    val totalPrice: Long,
    val orderStatus: OrderStatus,
    val createdAt: LocalDateTime,
    val items: List<OrderItemResponse>,
    )
